kitchen/ph5tostationxml.py

```diff
@@ -401,7 +401,7 @@ def create_obs_channel(self, station_list, deployment, cha_code, loc_code,
         if self.ph5.Das_t.has_key(das):
             Das_t = ph5API.filter_das_t(self.ph5.Das_t[das]['rows'],
                                         station_list[deployment][0]['channel_number_i'])
-            Receiver_t=self.ph5.get_receiver_t (Das_t[0], by_n_i=True)
+            Receiver_t=self.ph5.get_receiver_t (Das_t, by_n_i=True)

```

Comment:
I'm not sure if this is right. I just made this change and now get and error.

http://service.iris.edu/ph5beta/station/1/query?network=8A&format=TEXT&level=CHANNEL
returns:
Error 500: list indices must be integers, not str